cheroot/test/test_core.py

```diff
@@ -19,10 +19,8 @@
 localDir = os.path.dirname(__file__)
 base_path = os.path.join(os.getcwd(), localDir)
 favicon_path = os.path.join(base_path, '../favicon.ico')
-
-favicon_ico_file = pathlib.Path(os.path.join(base_path, '../favicon.ico'))
-static_dir = pathlib.Path(os.path.join(base_path, 'static/'))
-index_html_file = pathlib.Path(os.path.join(base_path, 'static/index.html'))
+static_dir = os.path.join(base_path, 'static/')
+index_html_path = os.path.join(base_path, 'static/index.html')
```

Comment:
But at the line 34, `favicon_path` was referenced when set up the server. 
So either put `favicon_path` separately in different places, or remain being a global const.

Only there two variables as shown above can be moved to the test.